packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.tsx

```diff
@@ -210,3 +251,13 @@ export function QueryWrapper(props: QueryWrapperProps) {
     </QueryContextProvider>
   )
 }
+
+export function QueryWrapper(props: QueryWrapperProps) {
+  // Wrap in a Jotai provider to ensure the Jotai atomic state is unique to this component tree
+  // i.e. other instances of QueryWrapper will not share state with this instance
+  return (
+    <Provider>
+      <_QueryWrapper {...props} />
+    </Provider>
+  )
```

Comment:
I verified that multiple instances of QueryWrapper on the same page don't share state (e.g. ADKP Study Details Page has multiple QueryWrapperPlotNav instances)